### PR TITLE
feat: Added section for Azure AI with 3 recommendations

### PR DESCRIPTION
**Pull request by khushal08**
# Overview/Summary

Created a new section for Azure AI with first three recommendations

## As part of this pull request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/tree/main)
- [x] Ensured PR tests are passing
- [x] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: azure-resources/ArtificialIntelligence/AI/_index.md
Comment (ejhenry) [inline azure-resources/ArtificialIntelligence/AI/_index.md:1]: The directory structure needs to align with a valid resource type. For example, azure-resources/CognitiveServices/accounts

File: azure-resources/ArtificialIntelligence/AI/recommendations.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+- description: Ensure to deploy a pay as you go instance of the model when using provisioned throughput to manage overflow effectively
+  aprlGuid: 0c193899-da60-4a52-b4a0-77d75ac8c5c5
+  recommendationTypeId: null
+  recommendationControl: Resiliency
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/AI
```
Comment (ejhenry) [inline azure-resources/ArtificialIntelligence/AI/recommendations.yaml:6]: The value needs to be a valid resource type and should align with the directory within which the recommendation is located.

File: azure-resources/ArtificialIntelligence/AI/recommendations.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  learnMoreLink:
+    - name: Learn More
+      url: "https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/provisioned-throughput"
+
+- description: Ensure that models are deployed using Global batch for large scale processing
+  aprlGuid: 8aa9744b-f302-4b05-9776-51d6dd3d0c3a
+  recommendationTypeId: null
+  recommendationControl: Scalability
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/AI
```
Comment (ejhenry) [inline azure-resources/ArtificialIntelligence/AI/recommendations.yaml:23]: The value needs to be a valid resource type and should align with the directory within which the recommendation is located.

File: azure-resources/ArtificialIntelligence/AI/recommendations.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  learnMoreLink:
+    - name: Learn More
+      url: "https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/deployment-types#global-batch"
+
+- description: Ensure AOAI models are deployed using Data Zone Standard for data residency requirements
+  aprlGuid: ac3add17-013e-41a5-af91-9fefce794a00
+  recommendationTypeId: null
+  recommendationControl: Governance
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/AI
```
Comment (ejhenry) [inline azure-resources/ArtificialIntelligence/AI/recommendations.yaml:40]: The value needs to be a valid resource type and should align with the directory within which the recommendation is located.

File: azure-specialized-workloads/ai/recommendations.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+- description: Ensure to deploy a pay as you go instance of the model when using provisioned throughput to manage overflow effectively
+  aprlGuid: 0c193899-da60-4a52-b4a0-77d75ac8c5c5
+  recommendationTypeId: null
+  recommendationControl: Resiliency
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/AI
```
Comment (ejhenry) [inline azure-specialized-workloads/ai/recommendations.yaml:6]: The value needs to be a valid resource type. If the recommendation does not directly align with a specific resource type, please us 'Microsoft.Subscription/subscriptions' for the value.

File: azure-specialized-workloads/ai/recommendations.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  learnMoreLink:
+    - name: Learn More
+      url: "https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/provisioned-throughput"
+
+- description: Ensure that models are deployed using Global batch for large scale processing
+  aprlGuid: 8aa9744b-f302-4b05-9776-51d6dd3d0c3a
+  recommendationTypeId: null
+  recommendationControl: Scalability
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/AI
```
Comment (ejhenry) [inline azure-specialized-workloads/ai/recommendations.yaml:23]: The value needs to be a valid resource type. If the recommendation does not directly align with a specific resource type, please us 'Microsoft.Subscription/subscriptions' for the value.

File: azure-specialized-workloads/ai/recommendations.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  learnMoreLink:
+    - name: Learn More
+      url: "https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/deployment-types#global-batch"
+
+- description: Ensure AOAI models are deployed using Data Zone Standard for data residency requirements
+  aprlGuid: ac3add17-013e-41a5-af91-9fefce794a00
+  recommendationTypeId: null
+  recommendationControl: Governance
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/AI
```
Comment (ejhenry) [inline azure-specialized-workloads/ai/recommendations.yaml:40]: The value needs to be a valid resource type. If the recommendation does not directly align with a specific resource type, please us 'Microsoft.Subscription/subscriptions' for the value.

**Review by ejhenry — COMMENTED** [review]
@khushal08 let me know if you have any questions about my comments.

File: azure-resources/ArtificialIntelligence/AI/_index.md
Comment (khushal08) [reply azure-resources/ArtificialIntelligence/AI/_index.md:1]: Agree. Changes pushed.

File: azure-resources/ArtificialIntelligence/AI/recommendations.yaml (hunk 1 above)
Comment (khushal08) [reply azure-resources/ArtificialIntelligence/AI/recommendations.yaml]: Agree. Changes pushed.

Comment (khushal08) [reply azure-resources/ArtificialIntelligence/AI/recommendations.yaml]: Agree. Changes pushed.

File: azure-specialized-workloads/ai/recommendations.yaml (hunk 4 above)
Comment (khushal08) [reply azure-specialized-workloads/ai/recommendations.yaml]: Agree. Changes pushed.

File: azure-specialized-workloads/ai/recommendations.yaml (hunk 6 above)
Comment (khushal08) [reply azure-specialized-workloads/ai/recommendations.yaml]: Not applicable.

**Comment by khushal08** [comment]
@ejhenry Modified based on the feedback

**Review by oZakari — APPROVED** [review]
LGTM

**Review by oZakari — APPROVED** [review]

**Review by ejhenry — APPROVED** [review]
LGTM

**Pull request merged**